lrpar/src/lib/parser.rs

```diff
@@ -839,7 +839,7 @@ pub enum ParseRepair<LexemeT: Lexeme<StorageT>, StorageT: Hash> {
 #[derive(Clone, Debug, PartialEq)]
 pub struct ParseError<LexemeT: Lexeme<StorageT>, StorageT: Hash> {
     stidx: StIdx,
-    lexeme: LexemeT,
+    lexeme: LexemeT, // Can get the span through lexeme.
```

Comment:
I agree with the comment, but we shouldn't leave it in. [Same with the same comment on `statetable.rs`.]